tests/operator_test.go

```diff
@@ -1374,9 +1374,15 @@ spec:
 			kv, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(kv)
 			Expect(err).ToNot(HaveOccurred())
 
+			By("Waiting for virt-handler component to be patched")
+			waitForKvWithTimeout(kv, 30)
+
 			By("Test that patch was applied to DaemonSet")
 			Eventually(fetchVirtHandlerCommand, 60*time.Second, 5*time.Second).Should(ContainSubstring(maxDevicesCommandArgument))
 
+			By("Waiting for virt-handler DS to become ready")
+			verifyVirtHandlerIsReady(virtClient, kv, 60)
```

Comment:
nit: I see a contradiction here of what the `By` says to what the name of the function tells me. Maybe we should rename the function to be similar to `waitForKvWithTimeout`, i.e. `waitForVirtHandlerReadyWithTimeout`. WDYT?

Reply:
@dhiller Thanks! fixed